src/QmlControls/CommonTopMenu.qml

```diff
@@ -0,0 +1,249 @@
+import QtQuick.Dialogs  1.3
+
+import QGroundControl                       1.0
+import QGroundControl.Controls              1.0
+import QGroundControl.Palette               1.0
+import QGroundControl.MultiVehicleManager   1.0
+import QGroundControl.ScreenTools           1.0
+import QGroundControl.Controllers           1.0
+import QtGraphicalEffects                   1.12
+
+Rectangle {
+    id:     root
+    color: Qt.rgba(0, 0, 0, 0.4)
+    z : 3
+    property int currentToolbar: flyViewToolbar
+
+    readonly property int flyViewToolbar:   0
+    readonly property int planViewToolbar:  1
+    readonly property int simpleToolbar:    2
+
+    property var    _activeVehicle:     QGroundControl.multiVehicleManager.activeVehicle
+    property bool   _communicationLost: _activeVehicle ? _activeVehicle.vehicleLinkManager.communicationLost : false
+    property color  _mainStatusBGColor: qgcPal.brandingPurple
+
+    property var rootWidth: null;
+    property var rootHeight: null;
+
+    Row {
+        id:                     viewButtonRow
+        anchors.top:            parent.top
+        anchors.bottom:         parent.bottom
+        width: parent.width
+        anchors.fill: parent
+
+        //Logo Button
+        Button {
+            width: viewButtonRow.width * 0.115
+            height: parent.height
+            background: Rectangle {
+                color: "transparent"
+                Image {
+                    source: "/res/TopMenu_homebutton.svg"
+                    anchors.centerIn: parent
+                    width: 80
+                    height: parent.height - 10
+                }
+            }
+            onClicked: {
+                console.log("Logo btn Click");
+            }
+        }
+        //Move to MainPage Button
+        Button {
+            width: viewButtonRow.width * 0.19
+            height: parent.height
+            clip: true
+            background: Rectangle {
+                color: "#3d71d7"
+                Row {
+                    anchors.fill: parent
+                    Rectangle {
+                        height: parent.height - 32
+                        anchors.verticalCenter: parent.verticalCenter
+                        Row {
+                            anchors.fill: parent
+                            anchors.leftMargin: 28
+                            spacing: 15
+                            Image {
+                                source: "/res/TopMenu_Manual Flight.svg"
+                                height: parent.height
+                                width: height
+                            }
+                            Column {
+                                spacing: 10
+                                Text {                                    
+                                    font.pixelSize: ScreenTools.defaultFontPixelHeight * 1.8
+                                    text: qsTr("Intelligent Photo")
+                                    color: "white"
```

Comment:
Define a centralized constants for colors to ensure consistency and ease of updates.